plugintoolbox/dropper_script_wrapper_template.py

```diff
@@ -0,0 +1,41 @@
+from pathlib import PurePath
+
+AGENT_BINARY_WRAPPER_SCRIPT_TEMPLATE = b"""#!/bin/bash
```

Comment:
I feel we should include "bash" in the name.
```suggestion
AGENT_BINARY_WRAPPER_BASH_SCRIPT_TEMPLATE = b"""#!/bin/bash
```

Reply:
I think it is explanatory enough. And since it is constant I don't think it matters that much.

Reply:
Anyone new to Infection Monkey doesn't know what kind of script this is unless they look into the code/if we specify it in our plugin SDK documentation (and they read it). Adding bash to the name helps that.

Plus, what happens when we introduce a new kind of script template?